index.html

```diff
@@ -14,6 +14,7 @@ <h1>Acha's first commit!</h1>
     <h1>Reina's first commit!</h1>
     <h1>Reina's second commit!</h1>
     <h2>Acha's second commit!</h2>
-    <h2>Kwon's 2nd commit</h2>
+    <h2>Kwon's second commit</h2>
+    <button>ボタン</button>
```

Comment:
ボタンにlarge-btnクラスをつけませんか？
```suggestion
<button class="large-btn"></button>
```